src/videojs-http-streaming.js

```diff
@@ -789,7 +791,22 @@ class VhsHandler extends Component {
       },
       bandwidth: {
         get() {
-          return this.masterPlaylistController_.mainSegmentLoader_.bandwidth;
+          let bandwidthEst = this.masterPlaylistController_.mainSegmentLoader_.bandwidth;
+
+          const networkInformation = window.navigator.connection || window.navigator.mozConnection || window.navigator.webkitConnection;
+
+          if (this.options_.useNetworkInformationApi && networkInformation) {
+            // downlink returns Mbps
+            // https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation/downlink
+            const effectiveBandwidthBitsPerSec = networkInformation.downlink * 1000 * 1000;
+
+            // downlink maxes out at 10 Mbps. In the event that the player estimates a bandwidth
+            // greater than 10 Mbps, use the larger value to ensure that high quality streams are not
+            // filtered out.
```

Comment:
We might want to explicitly check if the network API reported bandwidth and the player reported are both over 10mbps and only then use the player/vhs bandwidth estimate. The way that the code is written now, if the player/vhs estimates higher bandwidth we will always use that. I think that the network API should be prioritized instead.